packages/shared/src/components/checklist/SquadChecklistCard.tsx

```diff
@@ -49,27 +45,17 @@ const SquadChecklistCard = ({ squad }: { squad: Squad }): ReactElement => {
       title={isDone ? 'Good job! you nailed it. 🥳' : 'Get started with squads'}
       description={`${totalStepsCount} simple steps to Squad greatness!`}
       steps={steps}
-      onRequestClose={onRequestClose}
+      className="max-w-full border-0"
     />
   );
 
-  if (!sidebarRendered) {
-    return (
-      <Modal
-        className="!w-auto"
-        isOpen={isChecklistVisible}
-        kind={ModalKind.FlexibleCenter}
-        onRequestClose={onRequestClose}
-      >
-        {checklistElement}
-      </Modal>
-    );
-  }
-
   return (
     <InteractivePopup
+      isDrawerOnMobile
+      drawerProps={{ className: { drawer: 'pb-4' } }}
       position={InteractivePopupPosition.RightEnd}
-      className="rounded-none !bg-transparent"
+      className="flex w-full max-w-[21.5rem] justify-center rounded-none"
```

Comment:
Why do we need a custom max width here?

Reply:
We added the `w-full` - on laptop, this would unnecessarily consume the full page.